src/call_log/call_log.c

```diff
@@ -217,4 +217,27 @@ __wt_call_log_open_session(WT_SESSION_IMPL *session, int ret_val)
     return (0);
 }
 
+/*
+ * __wt_call_log_set_timestamp --
+ *     Print the call log entry for the set timestamp API call.
+ */
+int
+__wt_call_log_set_timestamp(WT_SESSION_IMPL *session, const char *config, int ret_val)
+{
+    WT_CONNECTION_IMPL *conn;
+    char config_buf[128];
+    char objectid_buf[128];
+
+    conn = S2C(session);
+
+    WT_RET(__call_log_print_start(session, "connection", "set_timestamp"));
+    WT_RET(__wt_snprintf(objectid_buf, sizeof(objectid_buf), "\"objectId\": \"%p\"", conn));
```

Comment:
Let's use underscore instead of camel case for consistency?

Reply:
Yeah sounds good, I'll change the other messages to reflect that.